README.md

```diff
@@ -244,10 +244,8 @@ to create a free account to add new tickets.
 
 ## Releasing
 
-1. Make sure version, changelog, etc. have been updated.
-1. Commit and tag with new version number: e.g. `v1.2.3`
-1. Push tag to Github: `git push upstream --tags` (where `upstream` is the remote name of the puppetlabs fork of this repo)
-1. Wait for Travis CI to test and push new release to Rubygems.
+1. Run the release_prep GitHub Action with the new version number, for example, 1.2.3. It will create a new pull request and merge it
```

Comment:
maybe add another step like 'please check the changelog and ensure all issues/prs have correct labels'?

Reply:
I have added labeller so any PR without label will fail it CI

Reply:
yes, but that will ensure that *any* label is present. At Vox Pupuli we learned that those are not always correct, so it makes sense to review it again. One example is: multiple submitted PRs for the same topic, every PR has the correct bugfix label, one gets merged, the others closed. In those cases we prefer it to ignore the closed PRs and replace their label with something like 'skip-changelog'.

Reply:
okay this is right, I have added the 2nd step